### PR TITLE
chore: disable string escaping in snapshots

### PR DESCRIPTION
**Pull request by SimenB**
## Description

Makes the snapshots more readable. We'll probably change this default in Jest 29, but since you already use `jest-snapshot-serializer-raw` in this repo, I'm thinking this is something you want now?

## Checklist

- [ ] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory).
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/main/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -58,6 +58,9 @@ module.exports = {
     "jest-snapshot-serializer-raw",
     "jest-snapshot-serializer-ansi",
   ],
+  snapshotFormat: {
+    escapeString: false,
+  },
```
Comment (SimenB) [inline jest.config.js:63]: this is the only change outside of `yarn test -u`

**Comment by fisker** [comment]
Wow, much better, just to confirm, the `` ` `` character still need escape, right?

**Comment by SimenB** [comment]
Yes, the entire snapshot string is wrapped in backticks, so any `` ` `` within that needs to be escaped

**Review by fisker — APPROVED** [review]

**Review by sosukesuzuki — APPROVED** [review]
Thank you for doing this!

**Pull request merged**